### PR TITLE
[lab-intro-to-sklearn] Ana G.

### PR DESCRIPTION
**Pull request by AnaAGG**

### CODE REVIEW

**Comment by olmocorell** [comment]
¡Hola Ana! 🚀 🐰 🏖️ 🍄 
Te cuento cosinas del lab.
En primer lugar, te dejo este artículo bastante interesante para interpretar los datos estadísticos de OLS [aquí](https://medium.com/@jyotiyadav99111/statistics-how-should-i-interpret-results-of-ols-3bde1ebeec01)
Por aquí: 
Captura de pantalla 2021-02-25 a las 11 17 08
Nos pedían el r2, bueno, en todos los ejercicios usamos el r2, el accuracy lo utilizaremos para los modelos de clasificación....aunque es verdad que el valor te sale parecido a lo que te debe salir y tengo que mirar por qué...
Hay una forma de sacarlo con sklearn y es:

```
from sklearn.metrics import r2_score
r2 = r2_score(y_test,y_pred)
```

Enhorabuena por hacer el bonus y entender cómo mejora nuestro modelo cuando nos quedamos con menos columnas, si esas columnas aportan más valor a los datos.
¡¡Muy buen trabajo!! Tu lab está de 10

ytorch-data-scientist-pandas-nunpym-mpy-sklearn-bug-data-matplotlib-54757344
